src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -99,8 +100,28 @@ internal LibraryViewController(Window dynamoView, ICommandExecutive commandExecu
 
             this.commandExecutive = commandExecutive;
             InitializeResourceStreams(dynamoViewModel.Model, customization);
+            dynamoWindow.StateChanged += DynamoWindowStateChanged;
+            dynamoWindow.SizeChanged += DynamoWindow_SizeChanged;
         }
 
+        //if the window is resized toggle visibility of browser to force redraw
+        private void DynamoWindow_SizeChanged(object sender, SizeChangedEventArgs e)
```

Comment:
does the browser redraw or refresh here?

Reply:
the browser image stays constant.